### PR TITLE
added: ability to update info tag for currently playing item from python

### PR DESCRIPTION
**Pull request by notspiff**
useful for streams and shit. ref https://forum.kodi.tv/showthread.php?tid=322889&pid=2674265#pid2674265

### CODE REVIEW

File: xbmc/interfaces/legacy/Player.h [hunk 1]
```diff
@@ -575,6 +575,24 @@ namespace XBMCAddon
...
+      /// @brief \python_func{ updateInfoTag(item) }
+      ///-----------------------------------------------------------------------
+      /// Update info labels for currently playing item.
+      ///
+      /// @param item ListItem with new info
+      ///
+      /// @throws Exception          If player is not playing a file
+      ///
+      updateInfoTag();
+#else
```
Comment (MartijnKaijser) [inline xbmc/interfaces/legacy/Player.h]: Please add
`       ///-----------------------------------------------------------------------`
`       /// @python_v18 New function added.`

Comment (notspiff) [reply xbmc/interfaces/legacy/Player.h]: done. also added example

**Comment by DaveTBlake** [comment]
Be nice to have similar functionality via JSON.

**Comment by notspiff** [comment]
jenkins build this please

**Comment by notspiff** [comment]
jenkins build this please

**Comment by notspiff** [comment]
jenkins build this please

**Comment by notspiff** [comment]
jenkins build this please

**Comment by notspiff** [comment]
failure is unrelated.

**Pull request merged**